armi/utils/__init__.py

```diff
@@ -572,42 +572,6 @@ def slantSplit(val, ratio, nodes, order="low first"):
     return X
 
 
-def runFunctionFromAllModules(funcName, *args, **kwargs):
```

Comment:
This is just totally unused, yes? Thus the removal?

Reply:
1. Yes, totally unused, even downstream.
2. It is wildly dangerous and a "bad idea", so I want it out.
3. I had never noticed it before. I only saw it because I was looking for things in the `/utils/` folder to add code coverage to.